src/components/pages/Registration/Registration.tsx

```diff
@@ -0,0 +1,205 @@
+import React, { useState } from 'react';
+import { makeStyles, Theme, createStyles } from '@material-ui/core/styles';
+import { Typography } from '@material-ui/core';
+import styles from './Registration.module.css';
+import FormControl from '@material-ui/core/FormControl';
+import InputLabel from '@material-ui/core/InputLabel';
+import OutlinedInput from '@material-ui/core/OutlinedInput';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import IconButton from '@material-ui/core/IconButton';
+import Visibility from '@material-ui/icons/Visibility';
+import VisibilityOff from '@material-ui/icons/VisibilityOff';
+import Button from '@material-ui/core/Button';
+import { Redirect } from 'react-router-dom';
+import {
+  REACT_APP_GLIFIC_REGISTRATION_API,
+  REACT_APP_GLIFIC_AUTHENTICATION_API,
+} from '../../../config/axios';
+import clsx from 'clsx';
+import axios from 'axios';
+
+export interface RegistrationProps {}
+
+export const Registration: React.SFC<RegistrationProps> = () => {
+  const [password, setPassword] = useState('');
+  const [userName, setUserName] = useState('');
+  const [phoneNumber, setphoneNumber] = useState('');
+  const [confirmPassword, setconfirmPassword] = useState('');
+  const [showPassword, setshowPassword] = useState(false);
+  const [showConfirmPassword, setshowConfirmPassword] = useState(false);
+  const [authToken, setauthToken] = useState('');
+  const [authCode, setauthCode] = useState();
```

Comment:
minor thing but can we also update the other methods here? `setPhoneNumber`, `setConfirmPassword`, `setShowPassword` etc...